debian/vyos-1x.postinst

```diff
@@ -221,11 +221,10 @@ fi
 # Remove unwanted daemon files from /etc
 # conntackd
 # pmacct
-# fastnetmon
 # ntp
 DELETE="/etc/logrotate.d/conntrackd.distrib /etc/init.d/conntrackd /etc/default/conntrackd
         /etc/default/pmacctd /etc/pmacct
-        /etc/networks_list /etc/networks_whitelist /etc/fastnetmon.conf
+        /etc/networks_list /etc/networks_whitelist
```

Comment:
This is a part of the fastnetmon package and could be removed
There were options for fastnemon networks list to limit processing interesting destination prefixes.

Reply:
Removed that line.